lib/puppet_litmus/serverspec.rb

```diff
@@ -154,7 +154,9 @@ def run_bolt_task(task_name, params = {})
                 end
     result = OpenStruct.new(exit_code: exit_code,
                             stdout: result.first['result']['status'],
-                            stderr: result.first['result']['status'])
+                            stderr: result.first['result']['stderr'],
+                            version: result.first['result']['version'],
+                            _error: result.first['result']['_error'])
```

Comment:
rather than return a couple of values, could we return the whole bolt result hash, as a thing ?

Reply:
@tphoney thanks for the valuable comment.Sounds good.We can fetch any value from result hash.